packages/lintol/capstone/src/Models/DataResource.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Lintol\Capstone\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use App\User;
+
+class DataResource extends Model
+{
+    //
+    protected $fillable = [
+         'filename',
+         'url',
+         'filetype',
+         'status',
+         'stored',
+         'reportid'
```

Comment:
I take it `reportid` should be a nullable has-one to the Report object, rather than the other way around? If so, would be best to leave out this field and update the `reports` table